src/lib/scanner.ts

```diff
@@ -137,77 +132,25 @@ export class ScanRunner {
         });
     }
 
-    /**
-     *
-     * @param executionResult The results from a scan
-     * @returns The HTML-formatted scan results, or an empty string.
-     * @throws If {@code executionResult.result} is not a string.
-     * @throws If {@code executionResult.warnings} contains any warnings about methods not being found.
-     * @throws if {@code executionResult.status} is non-zero.
+        /**
+     * Uses the provided arguments to run a Salesforce Code Analyzer command.
+     * @param args The arguments to be supplied
      */
-    private processDfaResults(executionResult: ExecutionResult): string {
```

Comment:
Didn't see a need for this method now that we are getting a HTML directly from the runAction.